certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/service/converter/CertificateDataCheckboxDateRangeListConfigConverter.java

```diff
@@ -12,32 +15,38 @@
 public class CertificateDataCheckboxDateRangeListConfigConverter implements
     CertificateDataConfigConverter {
 
-  @Override
-  public ElementType getType() {
-    return ElementType.CHECKBOX_DATE_RANGE_LIST;
-  }
+    @Override
+    public ElementType getType() {
+        return ElementType.CHECKBOX_DATE_RANGE_LIST;
+    }
+
+    public CertificateDataConfig convert(ElementSpecification elementSpecification) {
+        if (!(elementSpecification.configuration() instanceof ElementConfigurationCheckboxDateRangeList configuration)) {
+            throw new IllegalStateException(
+                "Invalid value type. Type was '%s'".formatted(elementSpecification.configuration().type())
+            );
+        }
+        return CertificateDataConfigCheckboxDateRangeList.builder()
+            .text(configuration.name())
+            .label(configuration.label())
+            .hideWorkingHours(configuration.hideWorkingHours())
+            .previousDateRangeText(configuration.previousDateRangeText())
+            .min(date(configuration.min()))
```

Comment:
Maybe we need to update the tests since we switched to LocalDate from TemporalUnit

Reply:
I'm not sure i understand what should be updated in the tests. The expected result is LocalDate and the value to be converted is Period.